pulsar-client-admin/pom.xml

```diff
@@ -56,6 +56,11 @@
       <artifactId>jersey-media-json-jackson</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>jakarta.activation</groupId>
+      <artifactId>jakarta.activation-api</artifactId>
+    </dependency>
```

Comment:
Same as above, not sure why we need to add it here

Reply:
I have excluded `jakarta.activation-api` in the root pom.xml of the `jersey-media-json-jackson` dependency, some components depends on `jersey-media-json-jackson` will also exclude the `jakarta.activation-api`, In this PR we just make sure Pulsar using the consistent version of `jakarta.activation-api`, not remove the dependency. So add `jakarta.activation-api` dependency for the components which depends on `jersey-media-json-jackson` before. I'm not sure if we can remove it directly.